readthedocs/proxito/urls.py

```diff
@@ -86,6 +87,18 @@
     #     name='docs_detail',
     # ),
 
+    # External builds
+    # (requires to be before single version)
+    url(
+        (
+            r'^html/(?P<project_slug>{project_slug})/'
```

Comment:
If we want to have a nicer URL here, we need to change this code as well: https://github.com/readthedocs/readthedocs.org/blob/1876191b2b93c6e0d0965fa6503e06e39fe296c9/readthedocs/builds/models.py#L294-L303